docs/reference/changelog.rst

```diff
@@ -16,6 +16,17 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+.. _release-2.24.0:
+
+2.24.0 (unreleased)
+-------------------
+
+New features:
+
+- Add a new version of the ``PythonPlugin``. This new version is not registered by
```

Comment:
> `PythonPlugin`

For the apps, I prefer "Python plugin" in plain language. But for in-library docs (not common), I'm not so sure that's as useful. How do you feel about it?